core/server/master/src/main/java/alluxio/master/meta/AlluxioMasterRestServiceHandler.java

```diff
@@ -936,14 +949,22 @@ public Response getWebUIMetrics() {
           .setTotalBytesWrittenUfsThroughput(
               FormatUtils.getSizeFromBytes(bytesWrittenUfsThroughput));
 
+      //
+      // For the per UFS metrics below, if a UFS has been unmounted, its metrics will still exist
+      // in the metrics system, but we don't show them in the UI. After remounting the UFS, the
+      // new metrics will be accumulated on its old values and shown in the UI.
+      //
+
       // cluster per UFS read
       Map<String, String> ufsReadSizeMap = new TreeMap<>();
       for (Map.Entry<String, Gauge> entry : mr
           .getGauges((name, metric) -> name.contains(WorkerMetrics.BYTES_READ_UFS)).entrySet()) {
         alluxio.metrics.Metric metric =
             alluxio.metrics.Metric.from(entry.getKey(), (long) entry.getValue().getValue());
-        ufsReadSizeMap.put(metric.getTags().get(WorkerMetrics.TAG_UFS),
-            FormatUtils.getSizeFromBytes((long) metric.getValue()));
+        String ufs = metric.getTags().get(WorkerMetrics.TAG_UFS);
+        if (isMounted(ufs)) {
```

Comment:
@cheng-chang is it easy to add a test case?

Reply:
the existing test uses a lot of mocks, it's not easy to add a test for this.